scipy/signal/tests/test_windows.py

```diff
@@ -213,14 +213,21 @@ def test_windowfunc_basics():
         window(6, *params, sym=False)
 
 
-def test_needs_params():
```

Comment:
You have *replaced* this test with `test_kaiser_derived()`.  `test_needs_params` is actually a test of the `get_window` function, and I don't think it should be removed. If there is a good reason for removing it, that should be handled in a separate PR.